src/test/java/de/uhd/ifi/se/decision/management/jira/consistency/TestCipCalculation.java

```diff
@@ -0,0 +1,106 @@
+package de.uhd.ifi.se.decision.management.jira.consistency;
+
+import com.atlassian.jira.component.ComponentAccessor;
+import com.atlassian.jira.exception.CreateException;
+import com.atlassian.jira.issue.Issue;
+import com.atlassian.jira.issue.MutableIssue;
+import com.atlassian.jira.issue.link.IssueLinkType;
+import com.atlassian.jira.issue.link.IssueLinkTypeManager;
+import com.atlassian.jira.mock.issue.MockIssue;
+import com.atlassian.jira.project.Project;
+import com.atlassian.jira.user.ApplicationUser;
+import com.atlassian.jira.user.util.MockUserManager;
+import com.atlassian.jira.user.util.UserManager;
+import de.uhd.ifi.se.decision.management.jira.TestSetUp;
+import de.uhd.ifi.se.decision.management.jira.testdata.JiraIssues;
+import de.uhd.ifi.se.decision.management.jira.testdata.JiraProjects;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.ofbiz.core.entity.GenericEntityException;
+
+import java.sql.Timestamp;
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+public class TestCipCalculation extends TestSetUp {
+
+	private static Issue baseIssue;
+	private static List<MutableIssue> testIssues;
+	private static ApplicationUser user1, user2;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5455522)